### ISSUE TITLE
[feat req] Autotune and big room (echo) effects

### ISSUE THREAD

**Issue by matteoturini** [lyrebird-voice-changer/lyrebird#53]

**Comment by harrego** [comment]
Autotune is an infinite time scale feature I want too lol. And echo is being worked on in #97. Echo at least is coming soon!

**Comment by matteoturini** [comment]
> Autotune is an infinite time scale feature I want too lol.

Sorry, can you rephrase this? Does this mean it's not coming? What is an 'infinite time scale'?

**Comment by harrego** [comment]
Infinite timescale describes a feature that if time was infinite and long enough, it would eventually be implemented. It's something that I do want to add, but is low priority right now :)

**Comment by harrego** [comment]
However I was surprised to find https://github.com/ederwander/PyAutoTune which looks pretty good!
